examples/apps/generic_client/fabric_generic_client.py

```diff
@@ -604,9 +604,13 @@ def Main(args=None):
                 wo_receipt,
                 wo_params, jrpc_req_id)
         # Verify receipt signature
-        if generic_client.verify_receipt_signature(
-                retrieve_wo_receipt) is False:
-            logger.error("Receipt signature verification Failed")
+        if "result" in retrieve_wo_receipt:
```

Comment:
Is this check applicable only to fabric_generic_client and not required for generic_client direct model ?

Reply:
required for generic_client in direct model as well.